### ISSUE TITLE
Gibbsnet issue

### ISSUE THREAD

**Issue by LukasMosser** [wiseodd/generative-models#44]
I've been running the new gibbsnet implementation, but I can't find it to converge with the given settings.

Do you have any examples on what the results should look like?

**Comment by wiseodd** [comment]
My point on that implementation is to get the intuition of GibbsNet, and how to quickly implement it, esp. given ALI/BiGAN implementation.
Given this intuition feel free to experiment with another architecture and hyperparams.

Cheers,
Agustinus

**Comment by LukasMosser** [comment]
Yes, I understand. 
I just wanted to point out that it seems like this doesn't converge though.
After 2000 iterations:
![200](https://user-images.githubusercontent.com/4195648/34341482-56435620-e990-11e7-9ac8-c913aaad30b7.png)

The loss of the generator is very high (18) and discriminator loss goes to zero.
Have you gotten it to work with your code? I am using the code as is, no modifications.

**Comment by wiseodd** [comment]
Yeah, it's also what I experienced. It won't generate good result and the loss is blowing up.
I suspect we need to have stronger networks + much lower learning rate.

However, as I'd like to have everything in this repo with very simple architecture, I decided to push it as is :)

**Comment by LukasMosser** [comment]
I tried an 8 layer MLP for each network, as suggested in the paper with lr of 1e-5. Still no convergence.
Would be nice to get a version working. If I make any progress I'll update.
